tests/Integration/Behaviour/Features/Context/CartRuleFeatureContext.php

```diff
@@ -87,6 +87,14 @@ public function thereIsACartRuleWithNameAndPercentDiscountOf50AndPriorityOfAndQu
         $this->createCartRule($cartRuleName, $percent, 0, $priority, $cartRuleQuantity, $cartRuleQuantityPerUser);
     }
 
+    /**
+     * @Given /^there is a cart rule named "(.+)" that applies no discount with priority (\d+), quantity of (.*) and quantity per user (.*)$/
```

Comment:
Quantity are integer no?
```suggestion
     * @Given /^there is a cart rule named "(.+)" that applies no discount with priority (\d+), quantity of (\d+) and quantity per user (\d+)$/
```